res/css/style.css

```diff
@@ -36,6 +36,10 @@ body,
   z-index: 0;
 }
 
+#root-screenshot {
+  z-index: 14;
```

Comment:
why `14` while it could be `1` ? :)

Reply:
I was just setting it to a very high value without putting much thought behind that.

I will change that. :)